modules/aws/config.go

```diff
@@ -0,0 +1,30 @@
+package aws
+
+import (
+	"log"
+	"os"
+)
+
+var customEndpoints map[string]string
+
+// SetAwsEndpointsOverrides function to override the aws endpoints globall one time
+func SetAwsEndpointsOverrides(endpoints map[string]string) {
+
+	if HasAwsCustomEndPoints() {
+		log.Println("The aws custom endpoints have been already setup, check your configuration. The global endpoint override is just allow one time")
+		os.Exit(1)
```

Comment:
after giving this more thought, do we have to exit?  why not just return an error?  calling exit() should only be done for un-recoverable failures and this is has a simple recovery: do nothing.

Reply:
oh, just realized people should call HasAwsCustomEndpoints() first... sure that works.

Reply:
Since this is still open, I will point out that `os.Exit()` skips `defer` calls which is definitely not helpful.   Probably should use `log.Panicln()` instead.